package.json

```diff
@@ -9,18 +9,18 @@
         "jvmArgs": "--add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.invoke=ALL-UNNAMED --add-opens=java.base/java.nio.channels.spi=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.util.concurrent=ALL-UNNAMED --add-opens=java.base/javax.net.ssl=ALL-UNNAMED"
     },
     "scripts": {
-        "api-layer": "concurrently --names \"GS,DS,AC,DC,ZO,CS,CG\" -c cyan,yellow,white,blue,green npm:gateway-service npm:discovery-service npm:api-catalog-service npm:discoverable-client npm:mock-services npm:caching-service npm:cloud-gateway",
-        "api-layer-ci": "concurrently --names \"GS,DS,AC,DC,ZO,CS,NS\" -c cyan,yellow,white,blue,green,red npm:gateway-service-thin npm:discovery-service-thin npm:api-catalog-service-thin npm:discoverable-client npm:mock-services npm:caching-service npm:onboarding-enabler-nodejs-sample-app",
-        "api-layer-core": "concurrently --names \"GW,DS,AC\" -c cyan,yellow,white npm:gateway-service npm:discovery-service npm:api-catalog-service",
-        "api-layer-thin": "concurrently --names \"GW,DS,AC,DC,ZO,CS\" -c cyan,yellow,white,blue,green npm:gateway-service-thin npm:discovery-service-thin npm:api-catalog-service-thin npm:discoverable-client npm:mock-services npm:caching-service",
+        "api-layer": "concurrently --names \"ZS,DS,AC,DC,ZO,CS,CG\" -c cyan,yellow,white,blue,green npm:zaas-service npm:discovery-service npm:api-catalog-service npm:discoverable-client npm:mock-services npm:caching-service npm:cloud-gateway",
+        "api-layer-ci": "concurrently --names \"GS,DS,AC,DC,ZO,CS,NS\" -c cyan,yellow,white,blue,green,red npm:zaas-service-thin npm:discovery-service-thin npm:api-catalog-service-thin npm:discoverable-client npm:mock-services npm:caching-service npm:onboarding-enabler-nodejs-sample-app",
```

Comment:
missing ZS?

Reply:
yes I will upgrade also the others, I forgot to